content/en/events/2024/kcsna/_index.md

```diff
@@ -31,7 +31,9 @@ slack channel.
 
 ### Call for Session Proposals
 
-A CFP for sessions will be opened at a later point in time.
+A [CFP](https://forms.gle/5RJBiKtuYsBDYAsVA) for sessions will be open through Sunday, September 15th.
+
+[CFP]: (https://forms.gle/5RJBiKtuYsBDYAsVA)
```

Comment:
I'd pick one or the other rather than both, but I don't know that the build system actually cares. Thanks for the update!

```suggestion
A [CFP] for sessions will be open through Sunday, September 15th.

[CFP]: (https://forms.gle/5RJBiKtuYsBDYAsVA)
```

or

```
A [CFP](https://forms.gle/5RJBiKtuYsBDYAsVA) for sessions will be open through Sunday, September 15th.
```

Reply:
Ok, let me update. I haven't done a ton with markdown like this and I saw this in a few of the previous files and based this on that.